packages/bot-web-ui/src/stores/load-modal-store.ts

```diff
@@ -177,18 +242,20 @@ export default class LoadModalStore {
         const { loadFile } = this.root_store.google_drive;
         const { xml_doc, file_name } = await loadFile();
         load({ block_string: xml_doc, file_name, workspace: Blockly.derivWorkspace, from: save_types.GOOGLE_DRIVE });
-        this.toggleLoadModal();
+        //this.toggleLoadModal();
     }
 
     @action.bound
-    onEntered() {
+    onEntered = (): void => {
         if (this.tab_name === tabs_title.TAB_RECENT && this.selected_strategy) {
             this.previewRecentStrategy(this.selected_strategy.id);
+        } else {
+            this.previewRecentStrategy(this.selected_strategy.id);
```

Comment:
what is the differemce between this line and the line above? they are the same

Reply:
the above line is used for the popup and the below will be used to select the a strategy when the user logins to select the bot preview by default

Reply:
this condition runs the same thing any way .. either this condition is true (this.tab_name === tabs_title.TAB_RECENT && this.selected_strategy) or anything else it runs ` this.previewRecentStrategy(this.selected_strategy.id);`
so what the condition means here?

Reply:
done ✅